### ISSUE TITLE
Staging Review finding: Focus is not managed on click of continue

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#121916]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Intake Optimization - Agile Six Team
**Product:** Marital Status Questionnaire (VA Form 21P-0537)
**Feature:** 21P-0537 Form Digitization

## Findings details
**VA.gov Experience Standard - issue:** [User experience is not consistent with the VA.gov design principles.](https://design.va.gov/about/experience-standards/#users-experience-is-consistent-with-the-vagov-design-principles)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

From Step 1 of 4, to Step 2 of 4, clicking Continue, focus remains on the Continue button.

<details><summary>Link, screenshot or steps to recreate</summary>

Steps to recreate:
- Using keyboard only, after filling out the form on Step 1 of 4, TAB to Continue
- Hit ENTER
- Note where focus is when Step 2 of 4 loads (Deceased Veteran's name page)
- Note: this also occurs:
  - For almost all pages throughout the flow, except for the summary page
  - Using "Back" navigating to the same pages

<img width="535" height="205" alt="focus not managed" src="https://github.com/user-attachments/assets/42f5583c-b1f0-460a-984b-58035358ed25" />

</details>

## Recommended action

Ensure focus is managed correctly for SPA pages.

https://design.va.gov/templates/forms/accessibility-guidelines#managing-focus-in-form-flows

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 2.4.3 Focus Order - Level A](https://www.w3.org/TR/WCAG22/#focus-order)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by amponce** [comment]
We looked into this issue and added notes to the ticket here: https://github.com/department-of-veterans-affairs/va.gov-team/issues/122172#issuecomment-3423862852
